packages/flutter_tools/test/commands.shard/hermetic/generate_localizations_test.dart

```diff
@@ -319,4 +319,33 @@ untranslated-messages-file: lib/l10n/untranslated.json
     FileSystem: () => fileSystem,
     ProcessManager: () => FakeProcessManager.any(),
   });
+
+  testUsingContext('nullable-getter defaults to true', () async {
+    final File arbFile = fileSystem.file(fileSystem.path.join('lib', 'l10n', 'app_en.arb'))
+      ..createSync(recursive: true);
+    arbFile.writeAsStringSync('''
+{
+  "helloWorld": "Hello, World!",
```

Comment:
is this testing nullable getters?

Reply:
I'm not sure if I understand your question.

The `helloWorld` string is basic string to get the localizations file to generate.

Then we test that the generated file has
```
static AppLocalizations? of(BuildContext context)
```
// With nullable-getter true
instead of
```
// With nullable-getter false
static AppLocalizations of(BuildContext context)
```

Reply:
Ahh ok, thanks!